src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $border-width;
+  padding: $padding;
```

Comment:
The variable `$padding` is used here, but it's not defined within this file. Ensure that it is defined in an imported SCSS file or within this file.